src/confcom/azext_confcom/tests/latest/test_confcom_scenario.py

```diff
@@ -575,42 +575,8 @@ def test_image_layers_python(self):
             aci_policy.populate_policy_content_for_all_images()
             layers = aci_policy.get_images()[0]._layers
             expected_layers = [
-                "254cc853da6081905c9109c8b9d99c9fb0987ba1d88f729088903cffb80f55f1",
-                "a568f1900bed60a0641b76b991ad431446d9c3a344d7b261f10de8d8e73763ac",
-                "c70c530e842f66215b0bd955877157ba24c3799303567c3f5673c45663ea4d15",
-                "3e86c3ccf1642bf584de33b49c7248f87eecd0f6d8c08353daa36cc7ad0a7b6a",
-                "1e4684d8c7caa74c6524172b4d5a159a10887613ed70f18d0a55d05b2af61acd",
-            ]
-            self.assertEqual(len(layers), len(expected_layers))
-            for i in range(len(expected_layers)):
-                self.assertEqual(layers[i], expected_layers[i])
-
-    def test_image_layers_nginx(self):
-        custom_json = """
-        {
-            "version": "1.0",
-            "containers": [
-                {
-                    "containerImage": "nginx:1.22",
-                    "environmentVariables": [],
-                    "command": ["echo", "hello"]
-                }
-            ]
-        }
-        """
-        with load_policy_from_str(custom_json) as aci_policy:
-            # pull actual image to local for next step
-            aci_policy.pull_image(aci_policy.get_images()[0])
-            aci_policy.populate_policy_content_for_all_images()
-            layers = aci_policy.get_images()[0]._layers
-
-            expected_layers = [
-                "5250e7d2517bcae4d264c84d8e7c6da14607ce867e29a81bf4327ee6896218a3",
-                "b6d54ad6a7223dd687d308c8562aaa7dfef2f5a88ec701fb3f89e49312832b82",
-                "8608c5be3af25ed58b2291999fe76cc021ced0ea70b6387c4373c6551f4d6ddb",
-                "1e0878890d701c494c8aeade31d15eaaf9b9c382c27e2519727cb5d1e91df764",
-                "233b6e2f8931a4d67930ac602688acc16c930926fcadc9e31195440db0737791",
-                "1053a7714644b99537bc0e8058a7e4771d2fe679ef54097e128a813f3c80a9cf",
+                "7b1ae25401fd6ca6c47b8b40aea586716a5097f61319e56fe362b3fc75e23d4b",
```

Comment:
these layers might get out of date if the 3.9-noroot image is later patched and keeps the same tag?

Reply:
that's kind of the point. It would be difficult to determine if it's a fault in the dmverity hashing or an image update, so having 2 different images with hardcoded layers helps us determine that it's a hashing issue. If only one of the tests fails, it's probably an image update.